src/com/lftechnology/java/training/niraj/addnumber/AddNumber.java

```diff
@@ -1,8 +1,6 @@
-
 package com.lftechnology.java.training.niraj.addnumber;
 
-import java.util.InputMismatchException;
-import java.util.Scanner;
+
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.InputMismatchException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)